src/SDKs/DataMigration/Management.DataMigration/Microsoft.Azure.Management.DataMigration.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataMigration</PackageId>
+    <Description>Provides developers with libraries for the Data Migration Service (DMS) under Azure Resource manager to deploy an instance of DMS. Create, Delete, Create project, Start Migration, and more.</Description>
+    <Version>0.1.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.Management.DataMigration</AssemblyName>
+    <PackageTags>Microsoft Azure resource management;Data Migration Service;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```

Comment:
@hitenjava add <packageReleaseNotes>
e.g. https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj

Reply:
Done.